js/preset.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 (function () {
-  var BLUR_VALUE = 3;
-  var BRIGHTNESS_VALUE = 2;
+  var MIN_BLUR = 0;
+  var MAX_BLUR = 3;
+  var MIN_BRIGHTNESS = 1;
+  var MAX_BRIGHTNESS = 2;
```

Comment:
я бы заменила
```suggestion
  var MAX_BRIGHTNESS = 3;
```
потому что filter: `brightness(1..3)`, и вычитала 1 при расчетах, чтобы константа не содержала в себе уже вычисленное значение